src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -542,7 +547,7 @@ public LicensedVideoSwapPageObject openLicensedVideoSwap(String wikiURL) {
   }
 
 
-  public void verifyUserLoggedIn(final String userName) {
+  public void :qverifyUserLoggedIn(final String userName) {
```

Comment:
illegale name
`Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.`